fapolicy_analyzer/ui/object_list.py

```diff
@@ -60,15 +60,14 @@ def wrap(x):
         matches = set(options).intersection(valueSet)
         return seperator.join([wrap(o) if o in matches else o for o in options])
 
-    def __colors(self, access, mode):
+    def __colors(self, access, status):
         green = (Colors.LIGHT_GREEN, Colors.BLACK)
         orange = (Colors.ORANGE, Colors.BLACK)
         red = (Colors.LIGHT_RED, Colors.WHITE)
         if access.upper() != "A":
             return red
 
-        numModes = len(set(mode.upper()).intersection({"R", "W", "X"}))
-        return green if numModes == 3 else orange if numModes > 0 else red
+        return green if status.lower() == "t" else orange
```

Comment:
You also need to look at the actual trust here, not just the trust status. something like this I think:
```suggestion
        elif trust.lower() in ["at", "st"] and status.lower() == "t":
            return green
        else:
            return orange
```